.github/workflows/basic.yml

```diff
@@ -25,6 +25,10 @@ jobs:
           override: true
           components: rustfmt, clippy
 
+      - name: Install protoc
+        run: |
+          apt-get update && apt-get install -y protobuf-compiler
```

Comment:
You might need `sudo` in front of these two commands.

Reply:
looks like it (with the `permission denied`'s). I had emulated https://github.com/confidential-containers/attestation-agent/blob/main/.github/workflows/eaa_kbc.yml#L36 which does not call out `sudo` :shrug: